homeassistant/brands/renson.json

```diff
@@ -0,0 +1,5 @@
+{
+  "domain": "renson",
+  "name": "Renson",
+  "integrations": ["renson", "renson_healthbox3"]
```

Comment:
Is it allowed to change the domain of the integration to renson_endura_delta? Is this not a breaking change?

Reply:
Changing the domain of an existing integration is a huge breaking change and should not be done.

Reply:
Sorry for asking, but is this a new rule that need to be follow? I ask this because I found out there are a lot of brands that have integrations with the same name as their domain.

Reply:
> Sorry for asking, but is this a new rule that need to be follow? 

Hmm... No, it's not a new rule, but I do see existing collisions as well. It looks like I'm wrong on this one. So 👍